src/components/Player.js

```diff
@@ -103,10 +103,19 @@ export default class Player extends Component {
 
     fullscreen.addEventListener(this.handleFullScreenChange)
     this.getMetadataTextTracks()
+
+    if (this.context?.setPlayer) {
+      const {player} = this.manager.getState()
+      this.context.setPlayer(player)
+    }
   }
 
   componentDidUpdate() {
     this.getMetadataTextTracks()
+    if (this.context?.setPlayer) {
+      const {player} = this.manager.getState()
+      this.context.setPlayer(player)
```

Comment:
this is relatively aggressive and updates the context whenever this component updates, but it's video player and constantly updating. probably more efficient approach exists.